### ISSUE TITLE
Strava Sauce Activity Sync broken

### ISSUE THREAD

**Issue by Nick2bad4u** [SauceLLC/sauce4strava#224]
Unable to sync any activites:
![image](https://github.com/user-attachments/assets/36262146-3b17-4321-8d63-aa989ab6f244)

[sauce-sync-138224227 (1).log](https://github.com/user-attachments/files/17962671/sauce-sync-138224227.1.log)

**Comment by mayfield** [comment]
Maybe an old version of sauce?

**Comment by Nick2bad4u** [comment]
> Maybe an old version of sauce?

Yup, I fixed it. the Edge version of sauce is way behind the chrome version in updates. Switched to sauce on chrome and boom, working.

Most edge addons always seem to be behind their google counterparts.
